Silicon/NXP/iMXPlatformPkg/Include/iMXVarStore.h

```diff
@@ -0,0 +1,20 @@
+/** @file
+*
+*  Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+*
+*  This program and the accompanying materials
+*  are licensed and made available under the terms and conditions of the BSD License
+*  which accompanies this distribution.  The full text of the license may be found at
+*  http://opensource.org/licenses/bsd-license.php
+*
+*  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+*
+**/
+
+#ifndef __IMX_VAR_STORE_H__
+#define __IMX_VAR_STORE_H__
+
+#define MAX_VARIABLE_SIZE 100
```

Comment:
SMBIOS 6.1.3 allows strings to be unlimited length, probably not a big issue but we can easily allow at least up to 16k-(size of meta data) which is the OP-TEE max variable size.